tests/samplers/test_logprobs.py

```diff
@@ -63,7 +63,8 @@ def test_get_prompt_logprobs(
         assert result.outputs[0].logprobs is not None
         assert len(result.outputs[0].logprobs) == max_tokens
         for logprobs in result.outputs[0].logprobs:
-            assert len(logprobs) == num_top_logprobs
+            assert (len(logprobs) == num_top_logprobs
+                    or len(logprobs) == num_top_logprobs + 1)
```

Comment:
nit: Better to add a comment explaining the "+1" here as well.

Reply:
Addressed in fded571c610608e54080b741a63bff0c998bdea6.